src/bdc/steps/google_places.py

```diff
@@ -20,7 +22,40 @@
 class GooglePlaces(Step):
     name = "Google_Places"
     URL = "https://maps.googleapis.com/maps/api/place/textsearch/json?query="
-    fields = ["business_status", "formatted_address", "name", "user_ratings_total"]
+
+    # fields that are expected as an output of the df.apply lambda function
+    df_fields = [
+        "place_id",
+        "business_status",
+        "formatted_address",
+        "name",
+        "user_ratings_total",
+        "rating",
+        "price_level",
+        "candidate_count_mail",
+        "candidate_count_phone",
+        "place_id_matches_phone_search",
+        "confidence",
+    ]
+
+    detailed_df_fields = ["website", "reviews", "type"]
```

Comment:
After the merge with the new dev make sure that this class still has a correct added_cols attribute.